sci-geosciences/gmt/gmt-5.4.4.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit bash-completion-r1 cmake-utils
+
+DESCRIPTION="Powerful map generator"
+HOMEPAGE="https://gmt.soest.hawaii.edu/"
+SRC_URI="mirror://gmt/${P}-src.tar.xz"
+
+LICENSE="GPL-2 gmttria? ( Artistic )"
+SLOT="5"
+KEYWORDS="~amd64 ~x86"
+IUSE="+dcw doc examples +fftw +gdal gmttria +gshhg htmldoc lapack openmp pcre pcre2 threads tutorial"
+
+RDEPEND="
+	!sci-biology/probcons
+	app-text/ghostscript-gpl
+	dcw? ( sci-geosciences/dcw-gmt-data )
+	fftw? ( sci-libs/fftw:3.0/3 )
+	gdal? ( sci-libs/gdal )
+	gshhg? ( sci-geosciences/gshhg-data )
+	>=sci-libs/netcdf-4.1[hdf5]
+	lapack? ( virtual/lapack )
+	pcre? ( dev-libs/libpcre )
+	pcre2? ( dev-libs/libpcre2 )
+	net-misc/curl
+	sys-libs/zlib"
+
+DEPEND="${RDEPEND}"
+
+REQUIRED_USE="
+	pcre? ( !pcre2 )
+	pcre2? ( !pcre )"
+
+src_prepare() {
+	cmake-utils_src_prepare
+	# Rename man pages to avoid a name conflict with gmt4
+	pushd man_release || die
+	local m c suffix newc
+	for m in *.gz; do
+		c=${m%%.*}
+		suffix=${m#*.}
+		# This man pages does'nt conflict
+		if [ "$c" == "gmt" ]; then continue; fi
+		if [ "$c" == "gmt.conf" ]; then continue; fi
+		if [ "$c" == "postscriptlight" ]; then continue; fi
+		newc=gmt_${c}
+		if [ "$c" == "gmt_shell_functions" ]; then newc=gmt5_shell_functions; fi
+		if [ "$c" == "gmtcolors" ]; then newc=gmt5colors; fi
+		mv "$c.$suffix" "$newc.$suffix" || die
+	done
+	popd || die
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DLICENSE_RESTRICTED="$(usex gmttria no yes)"
+		-DGMT_OPENMP="$(usex openmp)"
+		-DGMT_USE_THREADS="$(usex threads)"
+		-DGMT_INSTALL_TRADITIONAL_FOLDERNAMES=OFF # Install bash completions properly
+		-DGMT_INSTALL_MODULE_LINKS=OFF # Don't install symlinks on gmt binary, they are conflicted with gmt4
+		-DBASH_COMPLETION_DIR="$(get_bashcompdir)"
+		-DGMT_DATADIR="share/${P}"
+		-DGMT_DOCDIR="share/doc/${P}"
+		-DGMT_MANDIR="share/man"
+	)
+	use fftw || mycmakeargs+=( -DGMT_EXCLUDE_FFTW3=yes )
```

Comment:
Wouldn't inverted `usex` work here (as for LICENSE_RESTRICTED)?

Reply:
Did not understand what you mean here?

Reply:
`-DGMT_EXCLUDE_FFTW3=$(usex fftw no yes)` etc.

Reply:
(i.e. I'm wondering why your'e moving it out of the main group)